joomla/joomla-cms
# Make sure the fields are replaced in the fulltext too

### Summary of Changes

Make sure the fields are replaced in the fulltext too. cc @laoneo 

### Testing Instructions

1) create some fields for com_content
2) create an article with that fields
3) make sure you have an readmore tag in the article
4) display the article with an single article view.
5) notice that all fields are replaced
6) override (or change) this file components/com_content/views/article/tmpl/default.php at line 112 from 
```<?php echo $this->item->text; ?>``` to ```<?php echo $this->item->fulltext; ?>``` 
7) please notice that only everything after the readmore is displayed but the fields are **not** replaced.
8) apply this patch
9) notice that also the full text is replaced

### Expected result

Fields are also replaced in the fulltext 

### Actual result

Fields are not replaced in the fulltext

### Documentation Changes Required

none
